src/fromager/commands/build.py

```diff
@@ -85,19 +87,28 @@ def build_sequence(
     with open(build_order_file, "r") as f:
         for entry in progress.progress(json.load(f)):
             dist_name = entry["dist"]
-            dist_version = Version(entry["version"])
+            resolved_version = Version(entry["version"])
+            source_download_url = entry["source_url"]
+            req = Requirement(f"{dist_name}=={resolved_version}")
 
-            if skip_existing and _is_wheel_built(wkctx, dist_name, dist_version):
+            if skip_existing and _is_wheel_built(wkctx, dist_name, resolved_version):
                 logger.info(
                     "%s: skipping building wheels for %s==%s since it already exists",
                     dist_name,
                     dist_name,
-                    dist_version,
+                    resolved_version,
                 )
                 continue
 
-            logger.info("%s: building %s==%s", dist_name, dist_name, dist_version)
-            wheel_filename = _build(wkctx, dist_name, dist_version, sdist_server_url)
+            logger.info("%s: building %s==%s", dist_name, dist_name, resolved_version)
```

Comment:
Since sometimes we'll be downloading and not building, maybe change this to

```suggestion
            logger.info("%s: do not have wheel for %s==%s", dist_name, dist_name, resolved_version)
```

and then ensure that both `download_wheel()` and `_build()` log their action, too.

Reply:
I think all the other functions in `_build` log something related to their action so I don't think we need anything else there